collections/_posts/2022-11-23-Multi Channel Stochastic Variational Inference.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: review
+title: "Multi-Channel Stochastic Variationnal Inference"
```

Comment:
Normally, we use the full title of the article (even if it's long), as the title of the review.
```suggestion
title: "Multi-Channel Stochastic Variational Inference for the Joint Analysis of Heterogeneous Biomedical Data in Alzheimer's Disease"
```